.eslintrc.js

```diff
@@ -0,0 +1,180 @@
+// Standard Formatting rules for Kolbot scripts
+// Compatible with ESLint plugin
+
+module.exports = {
+	"root": true,
+	"extends": "eslint:recommended",
+	"parserOptions": {
+		"ecmaFeatures": {
+			"impliedStrict": true,
+		}
+	},
+	"env": {
+		"es6": true,
+	},
+	"globals": {
+		"global": true,
+		"include": true,
+		"print": true,
+		"me": true,
+		"td": true,
+		"getTickCount": true,
+		"delay": true,
+		"getParty": true,
+		"Pickit": true,
+		"takeScreenshot": true,
+		"getUnit": true,
+		"quit": true,
+		"clickMap": true,
+		"getBaseStat": true,
+		"clickItem": true,
+		"getCursorType": true,
+		"getPresetUnits": true,
+		"getDistance": true,
+		"copyUnit": true,
+		"getRoom": true,
+		"getLocaleString": true,
+		"DataFile": true,
+		"D2Bot": true,
+		"scriptBroadcast": true,
+		"isIncluded": true,
+		"FileTools": true,
+		"showConsole": true,
+		"getInteractedNPC": true,
+		"getDialogLines": true,
+		"getUIFlag": true,
+		"sendPacket": true,
+		"getPacket": true,
+		"getPath": true,
+		"rand": true,
+		"PresetUnit": true,
+		"getPresetUnit": true,
+		"getArea": true,
+		"getWaypoint": true,
+		"getScript": true,
+		"Room": true,
+		"say": true,
+		"load": true,
+		"addEventListener": true,
+		"getMercHP": true,
+		"Experience": true,
+		"checkCollision": true,
+		"gold": true,
+		"getLocation": true,
+		"login": true,
+		"sendCopyData": true,
+		"locationAction": true,
+		"getControl": true,
+		"debugLog": true,
+		"getCollision": true,
+		"transmute": true,
+		"submitItem": true,
+		"MuleLogger": true,
+		"createGame": true,
+		"joinGame": true,
+		"Line": true,
+		"removeEventListener": true,
+		"gidList": true,
+		"Unit": true,
+		"Party": true,
+		"UtilitySystem": true,
+		"CraftingSystem": true,
+		"moveNPC": true,
+		"ShitList": true,
+		"getPlayerFlag": true,
+		"clickParty": true,
+		"dopen": true,
+		"NTIPAliasClassID": true,
+		"Items": true,
+		"CustomConfig": true,
+		"getLists": true,
+		"pickItems": true,
+		"getNPCName": true,
+		"countItems": true,
+		"shopStuff": true,
+		"othersInGame": true,
+		"checkItem": true,
+		"Text": true,
+		"File": true,
+		"js_strict": true,
+		"handler": true,
+		"GameData": true,
+		"AreaData": true,
```

Comment:
Should these data modules be defined as global?

I know `GameData` is a module now -- so you should `const GameData = require('GameData');`.

Reply:
Sharp, thanks